bot/admin/server/src/main/kotlin/model/DialogsSearchQuery.kt

```diff
@@ -32,7 +32,8 @@ data class DialogsSearchQuery(
     val exactMatch: Boolean,
     val connectorType: ConnectorType?,
     val displayTests: Boolean = false,
-    val skipObfuscation: Boolean = false
+    val skipObfuscation: Boolean = false,
+    val rating: Set<Int>? = emptySet()
```

Comment:
Do not use nullable Set
`val rating: Set<Int> = emptySet()`

Reply:
resolved